torch/csrc/api/include/torch/nn/utils/clip_grad.h

```diff
@@ -57,6 +57,30 @@ inline float clip_grad_norm_(
   return clip_grad_norm_(params, max_norm, norm_type);
 }
 
+// Clips gradient of an iterable of parameters at specified value.
+// Gradients are modified in-place.
+// See https://pytorch.org/docs/stable/nn.html#clip-grad-value
+// for more details about this module.
+inline void clip_grad_value_(
+    std::vector<Tensor>& parameters,
+    float clip_value) {
+  std::vector<Tensor> params_with_grad;
+
+  for (const auto& param : parameters) {
+    auto& grad = param.grad();
+    if (grad.defined()) {
```

Comment:
We could write it as:
```suggestion
    if (param.grad().defined()) {
```
and remove `auto& grad = param.grad();`, since `grad` is only used once :D